torch/_refs/__init__.py

```diff
@@ -771,6 +793,22 @@ def _logical_or(a: TensorLikeType, b: TensorLikeType):
     aten_op=torch.ops.aten.logical_or,
 )
 
+
+def _logical_xor(a: TensorLikeType, b: TensorLikeType):
+    if not utils.is_boolean_dtype(a.dtype):
```

Comment:
Nice checking if the data is already boolean to avoid converting it, if possible